developer_manual/app/npm.rst

```diff
@@ -0,0 +1,76 @@
+===
+NPM
+===
+
+The node package manager ``npm`` is a widely used tool to manage JavaScript dependencies. In
+order to make the project setup easier for everybody, we recommend to follow the convention
+of typical scripts like ``build``, ``dev`` and ``test`` in the ``package.json`` file.
```

Comment:
shall we also add `lint`and `stylelint` ?

Reply:
yep, good idea :+1:

Reply:
Because this is separated behaviours I guess? 
I don't have such strong preference :)

Reply:
Me neither. Squash & merge?

Reply:
LETS GO! :muscle: